aiven/resource_project.go

```diff
@@ -60,6 +67,42 @@ var aivenProjectSchema = map[string]*schema.Schema{
 		Elem:        &schema.Schema{Type: schema.TypeString},
 		Optional:    true,
 	},
+	"cloud": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Target cloud",
+	},
+	"billing_currency": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Billing currency",
+	},
+	"available_credits": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Available credits",
+		Optional:    true,
+	},
+	"country": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Billing country",
+	},
+	"estimated_balance": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Estimated balance",
+	},
+	"payment_method": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Payment method",
+	},
+	"vat_id": {
+		Type:        schema.TypeString,
+		Computed:    true,
```

Comment:
This isn't computed but optional property the client can set.

Reply:
I will update go-client to include `vat_id` to create/update operations.